draft-packages/modal/KaizenDraft/Modal/Primitives/GenericModal.tsx

```diff
@@ -98,9 +98,11 @@ class GenericModal extends React.Component<GenericModalProps> {
   preventBodyScroll() {
     const hasScrollbar =
       window.innerWidth > document.documentElement.clientWidth
-    document.documentElement.classList.add(
-      ...[styles.unscrollable, hasScrollbar && styles.pseudoScrollbar]
-    )
+    const scrollStyles = [styles.unscrollable]
+    if (hasScrollbar) {
+      scrollStyles.push(styles.pseudoScrollbar)
+    }
+    document.documentElement.classList.add(...scrollStyles)
```

Comment:
This was generating a type error, as the styles applied could contain a boolean. I've changed to ensure it's always only `string[]`